egs/rm/s5/local/chain/tuning/run_tdnn_wsj_rm_1a.sh

```diff
@@ -0,0 +1,189 @@
+#!/bin/bash
+
+# This script uses weight transfer as a Transfer learning method
+# and use already trained model on wsj and removes the last layer and
+# add new randomly initialized layer and retrain the whole network,
+# while training new added layer using rm data.
+# The chain config is as in run_tdnn_5n.sh and the result is:
+#System tdnn_5n tdnn_wsj_rm_1a
+#WER      2.71     2.09
+set -e
+
+# configs for 'chain'
+stage=0
+train_stage=-10
+get_egs_stage=-10
+dir=exp/chain/tdnn_wsj_rm_1a
+
+# training options
+num_epochs=2
+initial_effective_lrate=0.005
+final_effective_lrate=0.0005
+leftmost_questions_truncate=-1
+max_param_change=2.0
+final_layer_normalize_target=0.5
+num_jobs_initial=2
+num_jobs_final=4
+minibatch_size=128
+frames_per_eg=150
+remove_egs=false
+xent_regularize=0.1
+
+# configs for transfer learning
+srcdir=../../wsj/s5   # base dir for source dataset.
+src_tdnn_affix=1d
```

Comment:
I prefer all things to be given explicitly and directly, e.g. src_mfcc_config, src_mdl, extractor, without making too many assumptions about the directory structure where things are coming from.  Document the meaning of all of these in the script you are calling (e.g. in the usage message and via comments when you declare the variables).  Check the ivector extractor id if feasible.